tests/components/rachio/test_config_flow.py

```diff
@@ -111,16 +111,22 @@ async def test_form_homekit(hass):
     await setup.async_setup_component(hass, "persistent_notification", {})
 
     result = await hass.config_entries.flow.async_init(
-        DOMAIN, context={"source": "homekit"}
+        DOMAIN,
+        context={"source": "homekit"},
+        data={"properties": {"id": "AA:BB:CC:DD:EE:FF"}},
     )
     assert result["type"] == "form"
     assert result["errors"] == {}
+    flows_in_progress = hass.config_entries.flow.async_progress()
```

Comment:
I had assumed it would always be the first flow since the test fixture is giving a clean version of `hass`.  Is it possible to have multiple flows in progress or flows come in from discovery with `async_test_home_assistant`?

Reply:
im not sure honestly. I just was looking to see prior art on best way to access context  when using core helpers to init the flow.

I dont think it hurts because it ensures the flow ids match and such. It might future proof the test who knows :)

It is just a suggestion so ill leave it to you if its worth the change.

Reply:
Works for me